pkg/controller/manager/manager_controller.go

```diff
@@ -146,7 +146,7 @@ func Add(mgr manager.Manager, opts options.AddOptions) error {
 	}
 	for _, namespace := range namespacesToWatch {
 		for _, secretName := range []string{
-			render.ManagerTLSSecretName, render.ElasticsearchManagerUserSecret,
+			render.ManagerTLSSecretName, render.ElasticsearchManagerUserSecret, relasticsearch.PublicCertSecret,
```

Comment:
Please include a comment about why the PublicCertSecret is being included. Without that, it is likely to be removed again in the future.